File: 8_Kyu_Kata/kata-example-twist/Python/kata_example_twist_test.py
# [object Object] - Kata Example Twist  [ ID: 525c1a07bb6dda6944000031  (kata-example-twist) ]
# URL: https://www.codewars.com/kata/525c1a07bb6dda6944000031
# Category: REFERENCE  |  Tags: FUNDAMENTALS
# ******************************************************************************
import codewars_test as test
from solution import websites

@test.describe("Fixed Tests")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(len(websites), 1000)
        test.assert_equals(type(websites), list)
        test.assert_equals(list(set(websites)), ["codewars"])

File: kata-8-kyu/kata-example-twist/javascript/kataExampleTwist.Test.js
//+ ====================================================================================================================
//+
//+ 8 kyu - Kata Example Twist  [ ID: 525c1a07bb6dda6944000031 ] (kata-example-twist)
//+ URL: https://www.codewars.com/kata/525c1a07bb6dda6944000031
//+ Category: REFERENCE  |  Tags: FUNDAMENTALS
//+
//+ ====================================================================================================================

const { expect } = require("chai")
const { websites } = require("./kataExampleTwist")

describe("Tests", () => {
  it("test", () => {
    expect(websites.length > 0, "The array is still empty")
    expect(websites.length == 1000, "The array does not equal 1,000")
    expect(
      websites.every(function (e) {
        return e == "codewars"
      }),
      'Every element in the array must contain the value "codewars"'
    )
  })
})
